Investigate to use `go mod` in Travis CI

`npm ci` seems to update `go.mod` file to use latest.
To use `go mod`, we need to investigate more.

BTW, Error about `src/app/backend/dashboard.go:158:26: undefined: prometheus.Handler` is caused due to prometheus.Handler() was removed in latest (1.0.0) prometheus/client-golang.

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
     "fix:license": "gulp update-license-headers",
     "clean": "rm -rf .go_workspace .tmp coverage dist npm-debug.log",
     "postversion": "node aio/scripts/version.js",
-    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false --versions.standalone=3.141.59 && go get -u github.com/golangci/golangci-lint/cmd/golangci-lint"
+    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false --versions.standalone=3.141.59 && curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.17.1"
   },
   "husky": {
     "hooks": {

